packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -106,6 +108,9 @@ export function BlockTypesTab( {
 		return result;
 	}, [ filteredItems, collections ] );
 
+	// hide block preview on unmount
+	useEffect( () => () => onUnmount(), [] );
```

Comment:
Shouldn't we just call `onHover( null )` here instead of a  new prop?

Reply:
Yes! Thank you @youknowriad. I was thinking at first to have a handler for `unMount`, but since it can be achieved internally, there is no need for the extra prop for now.